_posts/2022-10-26-basel-one-conference-report.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: 'BaselOne Conference Report'
+author: markus
+featuredImage: BaselOne22
+excerpt: "BaselOne 22 took place on Oct 19, 2022 in the Markthalle in Basel, Switzerland. Let us have a look back at this amazing event from different views: as one of the organizers, as speaker and as attendees."
+permalink: '/2022/10/26/baselone22.html'
+categories: [Java, Metrics, AI, NLP, Conferences, Kubernetes, Docker, DevOps, Istio, Service Mesh]
+header:
+  text: BaselOne22 Conference Report
+  image: post
+---
+
+BaselOne 2022 conference took place on October 19, 2022 in the Markthalle. As we were quite a few people from Karakun attending, be it as participants, speakers or organizer, we would like to share our impressions in this blog post. Some hard facts first:
+
+## Some figures about BaselOne22 
+BaselOne is a yearly event organized by the eponymous association existing of [Karakun AG](https://karakun.com), [Bluesky IT-Solutions AG](https://bluesky-it.ch) and [Baloise Insurance](https://baloise.com).
+
+In total, BaselOne 22 offered 26 talks by 29 speakers and 4 workshops to 160 paying attendees. All in all, almost 400 hours of organization time were spent to make it happen.
+
+
+
+## Day 1 - Conference Day
+
+As said, there were 26 talks, and it was challenging to choose which one to attend. The lineup was packed with well-known speakers like Ixchel Ruiz, Falk Sippach, Sandra Parsick, Otavio Santana, Katrin Rabow, Gerd Aschemann, Chris Engelbert and many more. The talks were nicely distributed across variety of topics and domains of IT. It is worth to mention that most of the talks had an introductory flavour to it to attract more beginners, but also to light a spark among more experienced folks. From more advanced talks to one of our colleague [George](https://twitter.com/grigala_) clearly stand out a talk about Service Mesh by [Abdellfetah Sghiouar](https://twitter.com/boredabdel):
+
+<blockquote class="twitter-tweet" style="justify-content: center"><p lang="en" dir="ltr">Very interesting talk by <a href="https://twitter.com/boredabdel?ref_src=twsrc%5Etfw">@boredabdel</a> <a href="https://twitter.com/Baselone?ref_src=twsrc%5Etfw">@Baselone</a> about Service Mesh with <a href="https://twitter.com/IstioMesh?ref_src=twsrc%5Etfw">@IstioMesh</a> <a href="https://twitter.com/kubernetesio?ref_src=twsrc%5Etfw">@kubernetesio</a> <a href="https://twitter.com/hashtag/BaselOne22?src=hash&amp;ref_src=twsrc%5Etfw">#BaselOne22</a> <a href="https://twitter.com/hashtag/BaselOne?src=hash&amp;ref_src=twsrc%5Etfw">#BaselOne</a> <a href="https://t.co/4Q2zZ2Vqaa">pic.twitter.com/4Q2zZ2Vqaa</a></p>&mdash; George Grigalashvili (@grigala_) <a href="https://twitter.com/grigala_/status/1582664710763053056?ref_src=twsrc%5Etfw">October 19, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
I personally don't like to load the script from twitter which als might be in conflict with data protection, what do the others think?

Reply:
no twitter for me

Reply:
Remove if so, I don't have any problem, just wanted to highlight things. I don't understand that data protection crap for the content that is publicly available.

Reply:
the data protection thing is that twitter spies on the user with that script. so if we load it from there embedd in out page we need to let the user know and decide if he is okay with that. does that help?

Reply:
Is it really true that Twitter can notice the IP of the visitor if the tweet is embedded in the page? I would doubt that. A visitor's IP is logged in Matomo (our web analytics) but why should it be forwarded to Twitter? 

Anyway, it is better to be cautious in this regard...

Reply:
Removed and adopted sentence per line rule for this paragraph. 

Will keep my extremist views regarding "data protection" nonsense for myself. 😁

Reply:
@kress04 it is as there is a request from the users browser to twitter.com when it loads the script. And as it is a JavaScript, it potentially logs a lot more than just the IP.

@grigala I often feel the same on the one hand, but on the other hand, data protection protects in most of the cases us as individuals so it´s worth to accept it ;-))